Mindee Invoice.ipynb

```diff
@@ -0,0 +1,417 @@
+    "print(currency)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "structured-briefing",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(433, 419) (532, 436)\n",
+      "(209, 489) (256, 508)\n",
+      "(426, 316) (488, 324)\n",
+      "(-1, 8) (142, 35)\n"
+     ]
+    }
+   ],
+   "source": [
+    "#now display the boxes on the image\n",
+    "data_dump = json.dumps(parsed_data.http_response)\n",
+    "data_json= json.loads(data_dump)\n",
+    "totalincl_x =data_json['document']['inference']['prediction']['total_incl']['polygon'][0][0]\n",
+    "totalincl_y =data_json['document']['inference']['prediction']['total_incl']['polygon'][0][1]\n",
+    "totalincl_x1=data_json['document']['inference']['prediction']['total_incl']['polygon'][2][0]\n",
+    "totalincl_y1=data_json['document']['inference']['prediction']['total_incl']['polygon'][2][1]\n",
+    "totalinclstart = (math.ceil(totalincl_x*width), math.ceil(height*totalincl_y))\n",
+    "totalinclend = (math.ceil(totalincl_x1*width), math.ceil(height*totalincl_y1))\n",
+    "print(totalinclstart, totalinclend)\n",
+    "totalexcl_x =data_json['document']['inference']['prediction']['total_excl']['polygon'][0][0]\n",
+    "totalexcl_y =data_json['document']['inference']['prediction']['total_excl']['polygon'][0][1]\n",
+    "totalexcl_x1=data_json['document']['inference']['prediction']['total_excl']['polygon'][2][0]\n",
+    "totalexcl_y1=data_json['document']['inference']['prediction']['total_excl']['polygon'][2][1]\n",
+    "totalexclstart = (math.ceil(totalexcl_x*width), math.ceil(height*totalexcl_y))\n",
+    "totalexclend = (math.ceil(totalexcl_x1*width), math.ceil(height*totalexcl_y1))\n",
+    "print(totalexclstart, totalexclend)\n",
+    "due_x =data_json['document']['inference']['prediction']['due_date']['polygon'][0][0]\n",
+    "due_y =data_json['document']['inference']['prediction']['due_date']['polygon'][0][1]\n",
+    "due_x1=data_json['document']['inference']['prediction']['due_date']['polygon'][2][0]\n",
+    "due_y1=data_json['document']['inference']['prediction']['due_date']['polygon'][2][1]\n",
+    "duestart = (math.ceil(due_x*width), math.ceil(height*due_y))\n",
+    "dueend = (math.ceil(due_x1*width), math.ceil(height*due_y1))\n",
+    "print(duestart, dueend)\n",
+    "supplier_x =data_json['document']['inference']['prediction']['supplier']['polygon'][0][0]\n",
+    "supplier_y =data_json['document']['inference']['prediction']['supplier']['polygon'][0][1]\n",
+    "supplier_x1=data_json['document']['inference']['prediction']['supplier']['polygon'][2][0]\n",
+    "supplier_y1=data_json['document']['inference']['prediction']['supplier']['polygon'][2][1]\n",
+    "supplierstart = (math.ceil(supplier_x*width), math.ceil(height*supplier_y))\n",
+    "supplierend = (math.ceil(supplier_x1*width), math.ceil(height*supplier_y1))\n",
+    "print(supplierstart, supplierend)"
```

Comment:
that cell is a bit long I think :sweat_smile:

Reply:
suggestions?  It's just doing the work for a bunch of variables all at once.

Reply:
Then the next cell will have to be changed to:
```python
plt.imshow(image2)
plt.show()
```

Reply:
This one is important for understandability I think